src/containers/Ledgers/LedgerEntryHashTrustedCount.tsx

```diff
@@ -0,0 +1,57 @@
+import { useTranslation } from 'react-i18next'
+import { useTooltip } from '../shared/components/Tooltip'
+import { Hash, ValidatorResponse } from './types'
+
+export const LedgerEntryHashTrustedCount = ({
+  hash,
+  unlCount,
+  validators,
+}: {
+  hash: Hash
+  unlCount?: number
+  validators: { [pubkey: string]: ValidatorResponse }
+}) => {
+  const { t } = useTranslation()
+  const { hideTooltip, showTooltip } = useTooltip()
+  const className = hash.trusted_count < (unlCount || 0) ? 'missed' : ''
+
+  const getMissingValidators = () => {
+    const unl = {}
+
+    Object.keys(validators).forEach((pubkey) => {
+      if (validators[pubkey].unl) {
+        unl[pubkey] = false
```

Comment:
Why do we set this to false instead of the unl value? It seems similar to `undefined` and I took a second to re-read to confirm that `false` just seems to be a sentinel value that's main trait is that it is NOT undefined?

Reply:
I honestly dont know this code was written this way before I just broke it out into its own component.   I did rewrite it in the streams refactor though to this 

```
const status = useMemo(() => {
    const missing = [...(unl || [])]
    validations.forEach((v) => {
      if (!validators?.[v.validation_public_key]) {
        return
      }

      const missingIndex = missing.findIndex(
        (assumedMissing) => assumedMissing === v.validation_public_key,
      )
      if (missingIndex !== -1) {
        missing.splice(missingIndex, 1)
      }
    })

    return {
      missing: missing.map((v) => validators?.[v]),
      trustedCount: (unl?.length || 0) - missing.length,
    }
  }, [unl, validations])
  ```